build.py

```diff
@@ -64,7 +65,13 @@
 # incorrectly load the other version of the openvino libraries.
 #
 TRITON_VERSION_MAP = {
-    '2.11.0dev': ('21.06dev', '21.04', '1.7.1', '2021.2.200', '2021.2.200')
+    '2.11.0dev':
+      ('21.06dev',   # triton container
+       '21.04',      # upstream container
+       '1.7.1',      # ORT
+       '2021.2.200', # ORT OpenVINO
+       '2021.2.200', # Standalone OpenVINO (non-windows)
+       '2021.2')     # Standalone OpenVINO (windows)
```

Comment:
How is 2021.2 different from 2021.2.200, does something stop us from using 2021.2.200 on windows?

Reply:
2021.2 is the actual branch that we build from and I will be moving linux to build from that branch as well in a separate change. Right now we are using OV from a prebuilt package that has version 2021.2.200... not sure where the 200 comes from.

Reply:
Eventually we need to get the 3 openvino entries down to 1 so that we are using the same OV version for everything

Reply:
200 is the build number of the release.